server/komodo-rest/src/main/java/org/komodo/servicecatalog/DataSourceDefinition.java

```diff
@@ -93,7 +93,13 @@ public boolean isResouceAdapter() {
     protected void ds(Properties props, DefaultServiceCatalogDataSource scd, String key, String value) {
         if (scd.getDefinition().isResouceAdapter()) {
             props.setProperty(
-                    "swarm.resource-adapter.resource-adapters." + scd.getName() + ".connection-definitions." + key,
+                "swarm.resource-adapters.resource-adapters." 
+                        + scd.getName() 
+                        + ".connection-definitions." 
+                        + scd.getName() 
+                        + ".config-properties."
+                        + key
+                        + ".value",
```

Comment:
Can you provide a quick explanation on
a) why this has changed;
b) why the much longer name
c) how this would impact both salesforce and other resource adapters
d) is this a breaking change?

Reply:
No, it is not breaking change, it was never correct to begin with. The properties were designed before publish operation. See example here https://github.com/rareddy/wildfly-swarm-teiid-examples for properties required to create a RA.